src/libraries/System.Private.CoreLib/src/System/Globalization/OrdinalCasing.Icu.cs

```diff
@@ -40,40 +41,46 @@ internal static partial class OrdinalCasing
         // s_casingTable is covering the Unicode BMP plane only. Surrogate casing is handled separately.
         // Every cell in the table is covering the casing of 256 characters in the BMP.
         // Every cell is array of 512 character for uppercasing mapping.
-        private static ushort []?[] s_casingTable =
+        private static ushort []?[]? s_casingTable;
+
+        // 0 - null
+        // 1 - s_noCasingPage
+        // The bits are in reverse order for readability, i.e. the highest order bit refers to
+        // the lowest index.
+        private static ReadOnlySpan<byte> s_casingTableInit => new byte[32]
         {
-            /* 0000-07FF */       s_basicLatin,            null,            null,            null,            null,            null,            null,            null,
```

Comment:
would be better to keep this table inside a comment. for easier reading and debugging.